controller/workitem.go

```diff
@@ -111,18 +136,44 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	if !authorized {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not authorized to access the space"))
 	}
+
+	if ctx.Payload.Data.Relationships != nil && ctx.Payload.Data.Relationships.BaseType != nil &&
```

Comment:
@jarifibrahim there should be one more check `ctx.Payload.Data.Relationships.BaseType.Data.ID != nil`

Reply:
We don't need `ctx.Payload.Data.Relationships.BaseType.Data.ID != nil` but we need `ctx.Payload.Data.Relationships.Basetype.Data != nil`

See https://github.com/fabric8-services/fabric8-wit/pull/2202/commits/765031f0248cbd2f6eb1e9d2ba01b17aa3d35c01